src/RestControllers/AuthorizationController.php

```diff
@@ -658,7 +666,7 @@ public function userLogin(): void
         }
         $continueLogin = false;
         if (isset($_POST['user_role'])) {
-            if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"], 'oauth2')) {
+            if (false && !CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"], 'oauth2')) {
```

Comment:
Looks like we still have the CSRF check blues :)

Bet there is a session thing going on when using the app within OpenEMR. Just let me know if this is only causing problems when using in OpenEMR or also causes problems when doing standard oauth2 flow outside OpenEMR and I'll then see if can identify this issue.

Reply:
Wow, good catch, can't believe I left that in there, no wonder when I was trying to reproduce the CSRF bug I wasn't hitting it anymore haha.

Reply:
Should be resolved now